src/simtools/simulator.py

```diff
@@ -75,6 +75,11 @@ def __init__(
         self._submit_options = self.args_dict.get("submit_options", None)
         self._extra_commands = extra_commands
 
+        self.sim_telarray_seeds = {
+            "seed": self.args_dict.get("sim_telarray_instrument_seeds"),
+            "random_instances": self.args_dict.get("sim_telarray_random_instrument_instances"),
+            "seed_file_name": "random_seeds.txt",  # name only; will be place in config dir
```

Comment:
This is the name I suggested to change `"random_seeds.txt"`. In production it won't make a difference, but in local running/testing, it could be useful to have different descriptive names.

Reply:
Changed.